builtin/logical/pki/path_revoke.go

```diff
@@ -104,6 +104,25 @@ func pathRotateCRL(b *backend) *framework.Path {
 	}
 }
 
+func pathRotateDeltaCRL(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `crl/rotate-delta`,
+
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.ReadOperation: &framework.PathOperation{
```

Comment:
*sigh* I wish this were a POST but I suppose it makes more sense to follow the crl/rotate way of things

Reply:
We could add both as Update operations if we wanted I suppose?